microcosm_pubsub/tests/test_result.py

```diff
@@ -40,7 +40,7 @@ def handler(message):
         result = MessageHandlingResult.invoke(
             handler=handler,
             message=self.message,
-        )
+        ).resolve(self.message)
```

Comment:
Rather to rewrite tests to validate these cases separately, I made the minimal change.